release/release_tests.yaml

```diff
@@ -344,3 +344,471 @@
     file_manager: job
 
   alert: xgboost_tests
+
+#######################
```

Comment:
I have a little concern this file will probably blow up... lol let's see how it will work